### PR TITLE
change the method of api request

### PR DESCRIPTION
**Pull request by chestnutchen**

### CODE REVIEW

File: src/mvc/BaseModel.js [hunk 1]
```diff
@@ -74,6 +74,31 @@ define(function (require) {
...
+     * 获取api请求，默认直接返回`this.requesters`
+     *
+     * @return {Object}
+     */
+    exports.getRequesters = function () {
+        return this.requesters;
+    };
+
+    /**
+     * 注册api，将api的名称添加`request`前缀，注册到model的属性下边
```
Comment (Justineo) [inline src/mvc/BaseModel.js]: 注册为model的**方法**

File: src/mvc/BaseModel.js [hunk 2]
```diff
@@ -74,6 +74,31 @@ define(function (require) {
...
+     */
+    exports.registerRequester = function () {
+        var me = this;
+        u.each(this.requesters, function (requester, reqName) {
```
Comment (Justineo) [inline src/mvc/BaseModel.js]: `this.getRequesters()`

Comment (Justineo) [inline src/mvc/BaseModel.js]: `reqName` -> `name` 吧，并没有其他东西

File: src/mvc/BaseModel.js [hunk 3]
```diff
@@ -74,6 +74,31 @@ define(function (require) {
...
+            me['request' + reqName.charAt(0).toUpperCase() + reqName.slice(1)] = requester;
```
Comment (Justineo) [inline src/mvc/BaseModel.js]: `'request' + u.pascalize(name)` 就好了吧

File: src/mvc/BaseModel.js [hunk 4]
```diff
@@ -74,6 +74,31 @@ define(function (require) {
...
+     * @return {Object}
+     */
+    exports.getRequesters = function () {
+        return this.requesters;
+    };
+
+    /**
+     * 注册api，将api的名称添加`request`前缀，注册到model的属性下边
+     */
+    exports.registerRequester = function () {
```
Comment (Justineo) [inline src/mvc/BaseModel.js]: `registerRequesters`

**Comment by leeight** [comment]
Can one of the admins verify this patch?

**Pull request merged**